ZenPacks/zenoss/ZenPackLib/lib/base/ComponentBase.py

```diff
@@ -49,6 +49,32 @@ class ComponentBase(ModelBase):
             }
         }
 
+    def __setattr__(self, name, value):
+        '''enforce type checking when setting _properties attributes'''
+        # dictionary of _property "type" keys and target Python classes
+        target_type_map = {'boolean': bool,
```

Comment:
The _lines_ type appears to be missing here. Is that intentional? I realize it would be slightly harder to handle than the other simple types, but verifying that it's a list of strings could be useful.